src/class-wpml-gutenberg-integration-factory.php

```diff
@@ -37,6 +37,11 @@ public function create() {
 			);
 
 			if ( is_admin() ) {
+				/** @var \WPML_Translation_Job_Factory $wpml_translation_job_factory */
+				global $wpml_translation_job_factory;
+
+				WPML\Container\share( [ $wpml_translation_job_factory, wpml_get_admin_notices() ] );
```

Comment:
@strategio I think these should be shared at the point where they are created and not here.

Reply:
Done @brucepearson:
- https://git.onthegosystems.com/wpml/sitepress-multilingual-cms/merge_requests/2393
- https://git.onthegosystems.com/wpml/wpml-translation-management/merge_requests/1395

This also had the consequence to display the notice immediately in CTE.
After a discussion with Amir, we agreed to show it always in post edit
or post list to make it consistent with ATE. That's why I added the
`set_restrict_to_screen_ids`.